Java/LFU.java

```diff
@@ -0,0 +1,144 @@
+package lfu;
+import java.util.LinkedList;
+import java.util.ListIterator;
+import java.util.HashMap;
+
+/**
+ * @author salman
```

Comment:
Could you suggest something in this reference? Something which can be suited just for this.